tests/test_busybox.py

```diff
@@ -48,6 +48,42 @@ def test_busybox_image_size(
     )
 
 
+@pytest.mark.parametrize(
+    "container_per_test", [BUSYBOX_CONTAINER], indirect=True
+)
+def test_busybox_links(container_per_test):
+    # Ensure all commands in /bin are links to /usr/bin/busybox
+    container_per_test.connection.run_expect(
+        [0],
+        'for i in /bin/*; do stat -c "%N" "$i" | grep "/usr/bin/busybox"; done',
+    )
+
+
+@pytest.mark.parametrize(
+    "container_per_test", [BUSYBOX_CONTAINER], indirect=True
+)
+def test_busybox(container_per_test):
+    # Test only a selection of all busybox tools
+    busybox = container_per_test.connection.run_expect([0], "busybox")
+    container_per_test.connection.run_expect([0], "true")
+    container_per_test.connection.run_expect(
+        [0], "echo 'test' | cat | grep 'test'"
+    )
+    assert "root" in container_per_test.connection.run_expect([0], "ps").stdout
+    assert (
+        "ORSXG5AK"
+        in container_per_test.connection.run_expect(
+            [0], "echo test | base32"
+        ).stdout
+    )
+    assert (
+        "dGVzdAo="
+        in container_per_test.connection.run_expect(
+            [0], "echo test | base64"
+        ).stdout
+    )
```

Comment:
I would suggest to split this into multiple test functions and give them more descriptive names. Also, none of these tests are performing any mutation, so you can use the `auto_container` fixture here instead.

Reply:
Good point. I moved all subtests to individual tests now. Please mark as resolved if you agree with my changes now.